bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/runtime/CodeCheckEnvironmentChecker.java

```diff
@@ -162,9 +162,9 @@ public String getLogFileLocation() {
      * @return The constructed analyze command.
      */
     public String createAnalyzeCommmand(String buildLog){
-        return codeCheckerCommand + " analyze " + getConfigValue(ConfigTypes.CHECKER_LIST) +
+        return "\"" + codeCheckerCommand + "\"" + " analyze " + getConfigValue(ConfigTypes.CHECKER_LIST) +
```

Comment:
Please do not mix a bugfix in the restructure PR. Please create a separate PR for that.

Reply:
@Vodorok will you do a separate PR for these changes too?